pkg/core/storelimit/limit.go

```diff
@@ -26,6 +26,8 @@ const (
 	AddPeer Type = iota
 	// RemovePeer indicates the type of store limit that limits the removing peer rate
 	RemovePeer
+	// SendSnapshot indicates the type of sending snapshot.
+	SendSnapshot
```

Comment:
Will it affect the TiKV threshold?

Reply:
No, It only affect the sending snapshot of every tikv. operator should be rejected if the origin's leader store has no available limit.